bit_stream_filter.go

```diff
@@ -0,0 +1,28 @@
+package astiav
+
+//#cgo pkg-config: libavcodec
+//#include <libavcodec/bsf.h>
+//#include <stdlib.h>
+import "C"
+import (
+	"unsafe"
+)
+
+// https://github.com/FFmpeg/FFmpeg/blob/release/5.1/libavcodec/bsf.h#L111
+type BitStreamFilter struct {
+	c *C.struct_AVBitStreamFilter
+}
+
+func newBitStreamFilterFromC(c *C.struct_AVBitStreamFilter) *BitStreamFilter {
+	if c == nil {
+		return nil
+	}
+	cc := &BitStreamFilter{c: c}
```

Comment:
Could you `return &BitStreamFilter{c: c}` right away without creating the intermediary `cc`?